modules/nw-ipfailover-virtual-ip-addresses-concept.adoc

```diff
@@ -6,7 +6,7 @@
 [id="nw-ipfailover-virtual-ip-addresses-concept_{context}"]
```

Comment:
It appears this module was removed from the assembly. Is it being used somewhere else? 
https://github.com/openshift/openshift-docs/pull/74064/files#diff-99eebb2772aa83a3592ed7e92101d539b046d16a056823eb8c968ad7b94b0788L55

Reply:
Oops! I intended to delete this after reworking the content. Thanks for catching this!